### PR TITLE
WIP: Style

### PR DESCRIPTION
**Pull request by olofvi**
## PT: https://www.pivotaltracker.com/story/show/151365572

My CSS is not working like I would like it to

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@import "bootstrap";
+//= require jquery3
+//= require popper
+//= require bootstrap-sprockets
```
Comment (tochman) [inline app/assets/stylesheets/application.scss]: ?? Is this really supposed to be here?

File: app/views/laundry_rooms/index.html.erb [hunk 2]
```diff
@@ -55,3 +55,7 @@
       <% end %>
     </table>
 <% end %>
+
+<%= month_calendar do |date| %>
+    <%= date.day %>
+<% end %>
```
Comment (tochman) [inline app/views/laundry_rooms/index.html.erb]: Please do not change anything in files that are not related to what you are doing. In this case you will end up with merge conflicts later. Focus on getting the navbar working

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -3,18 +3,32 @@
...
+      color: #633472;
+    h1 {
+      text-align: center;
+    }
+
+    h2 {
+      text-align: left;
+    }
+
+  </style>
```
Comment (tochman) [inline app/views/layouts/application.html.erb]: Your styling should be added to the scss file. Not the template.

**Review by tochman — CHANGES_REQUESTED** [review]
Restrain from making changes not related to the feature you are working on.

File: app/views/layouts/application.html.erb (hunk 3 above)
Comment (olofvi) [reply app/views/layouts/application.html.erb]: ok I move it

File: app/views/laundry_rooms/index.html.erb (hunk 2 above)
Comment (olofvi) [reply app/views/laundry_rooms/index.html.erb]: yeah I'll take it out

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (olofvi) [reply app/assets/stylesheets/application.scss]: maybe not

File: features/calendar.feature [hunk 4]
```diff
@@ -3,8 +3,13 @@ Feature: User can go to landing page
...
 
+  Background:
+    Given There is a user with email "unicorn@test.com" and password "password" in our database
+    And I am logged in as that user
+    And I navigate to the index page
+
   Scenario: User can navigate to calendar page
     Given there is a user logged in
-    And I navigate to the index
+    Then show me the page
```
Comment (tochman) [inline features/calendar.feature]: Please remove this

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -3,18 +3,21 @@
...
-<% flash.each do |message_type, message| %>
-    <%= content_tag(:div, message) %>
-<% end %>
-<%= yield %>
+<%= render 'shared/navigation' %>
+<div class="container">
+  <% flash.each do |key, value| %>
+      <div class="text-center <%= flash_class(key) %>">
+        <%= value %>
+      </div>
```
Comment (tochman) [inline app/views/layouts/application.html.erb:17]: Actually, the original syntax was better 
```
<%= content_tag(:div, value, class: ['text-center', flash_class(key)]) %>
```
😜

**Review by tochman — APPROVED** [review]

**Pull request merged**